invenio_assets/assets/build/webpack.config.js

```diff
@@ -202,16 +221,22 @@ var webpackConfig = {
     // Write manifest file which Python will read.
     new BundleTracker({
       path: config.build.assetsPath,
-      filename: path.join(config.build.assetsPath, "manifest.json") ,
+      filename: path.join(config.build.assetsPath, "manifest.json"),
       publicPath: config.build.assetsURL,
     }),
   ],
   performance: { hints: false },
+  snapshot: {
+    managedPaths: [],
+  },
+  watchOptions: {
+    followSymlinks: true,
```

Comment:
Can we add comments around, even link to GitHub issues, with explanations? Some of the options are obscure! :)

Reply:
it's on my to-do list, after a brief break on frontend :D